src/openads/application/OpenAds.js

```diff
@@ -50,6 +50,20 @@ export default class OpenAds {
     return this._container.getInstance({key: 'AddPositionUseCase'}).addPosition({id, name, source, placement, segmentation, sizes, native})
   }
 
+  /**
+   * Update a Position with given changes and refresh his Ad
+   * @param {string} id
+   * @param {object} position
+   * @param {string} position.name
+   * @param {string} position.placement
+   * @param {string} position.segmentation
+   * @param {Array} position.sizes
+   * @returns {Promise<Position>}
+   */
+  refreshPosition ({id, position}) {
```

Comment:
Why you ask? xD

Reply:
Because in method declaration there are only 2 params (id, position) but in jsdoc there are defined so much more, to be precise, some Position properties (name, sizes, ... )

Reply:
https://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc

Reply:
Exactly! I figured that out but I wanted it to be confirmed it was done on purpose.

Reply:
http://www.usaelputogoogle.com/

Reply:
#troll